Tabloid/client/src/components/TagManagment.js

```diff
@@ -0,0 +1,16 @@
+const baseUrl = 'api/Tag';
+
+export const getAllTags = () => {
```

Comment:
if this route is auth protected, you'll need to get an api token to pass in to this fetch call.

Reply:
@heymonicakay  Added auth to the tagmangement!